### PR TITLE
Update GitHub Actions workflow

### PR DESCRIPTION
**Pull request by m1ngyuan**
## PR Type

Refactoring 

## Describe what this PR does for and how you did.
Update GitHub Actions workflow

### Checklist
- [x] Code compiles correctly
- [ ] Pull Request has submit to 2020.0 and Greenwich
- [ ] Create at least one junit test if possible
- [x] All tests passing
- [ ] Extend documentation if necessary
- [ ] Add myself / the copyright holder to the AUTHORS file

### CODE REVIEW

**Comment by m1ngyuan** [comment]
Forward #265

File: spring-cloud-tencent-commons/src/test/java/com/tencent/cloud/common/util/ResourceFileUtilsTest.java [hunk 1]
```diff
@@ -35,7 +35,7 @@ public class ResourceFileUtilsTest {
 	@Test
 	public void testReadExistedFile() throws IOException {
 		String content = ResourceFileUtils.readFile("test.txt");
-		Assert.assertEquals("just for test\n", content);
+		Assert.assertEquals("just for test", content);
```
Comment (SkyeBeFreeman) [inline spring-cloud-tencent-commons/src/test/java/com/tencent/cloud/common/util/ResourceFileUtilsTest.java:38]: @lepdou 看下这里的更新是否符合要求。

Comment (lepdou) [reply spring-cloud-tencent-commons/src/test/java/com/tencent/cloud/common/util/ResourceFileUtilsTest.java:38]: 这里的话，好像 idea 会自动加换行符。如果case能跑过的话，改了也没问题的。

**Review by SkyeBeFreeman — APPROVED** [review]
LGTM.

**Comment by lepdou** [comment]
@m1ngyuan 

非常感谢~

**Comment by SkyeBeFreeman** [comment]
@m1ngyuan Can you cherry-pick this commit to branch 2020 and 2021?

**Pull request merged**